### DISCUSSION TITLE
User survey: What do you use Code App for?

### DISCUSSION THREAD

**Discussion by bummoblizard** [thebaselab/codeapp#1075] [Polls]
Hi everyone. I would like to know how you are using code so I can devote my efforts into developing related features.

**Comment by lucabae** [comment]
> Hi everyone. I would like to know how you are using code so I can devote my efforts into developing related features.

I use it to execute scripts on the terminal that I wrote on my PC, but I don't usually write code on here.

**Comment by antitheticverma** [comment]
Hi, just bought CodeApp for iPad to start working on mobile app development for learning purpose. But I've not progressed much as the code is not running without any error on terminal and I didn't find any similar issue already raised under issues.

**Comment by ghost** [comment]
I write tutorials and guides in Markdown and save everything to Github. This is the one Markdown editor I found easy to use that can directly commit to Github from my iPad. This is should a wonderful thing and makes my workflow so easier to maintain.

**Comment by NGBlock** [comment]
Hi, i want to use the app for various projects and create functional apps with gui but i‘m unsure if it‘s possible localy / with server. Other projects will probably just be some text-based games or console Applications.

(i‘m buying the app later today)

**Comment by TSPJ-DDW** [comment]
For a college student who doesn't have a lot of laptop battery life, this app has helped me tremendously! It makes it easier for me to do my studies anytime, anywhere!

**Comment by talaviram** [comment]
Others…
I really tried using it. It’s nicest thing for iPad OS. 
sadly, it never did svelte and also the limitations of iCloud API for having proper git while using Apple Playgrounds didn’t work out for me. 

But I still think it’s one of the most promising IDEs for the iPad.

**Comment by moon-cacke** [comment]
Code editing, text editing, code viewing

**Comment by joewen85** [comment]
i wish code app as a client. connect my develop server. all compute resource call server. about language serve, static check and .etc

**Comment by sightes** [comment]
i use this app for work on software development with ssh , great for get a coffe store and work , but still dont have ipynb compability

**Comment by mad-rees** [comment]
I've been using it for learning web development.

**Comment by Schinkennugget** [comment]
I use it to push stuff I wrote with another app to GitHub.

**Comment by Samuel525432** [comment]
It's just me 99% of the time me learning to code

I hope I'll learn c or python sometime this summer :3
